tests/integration/datasetio/test_datasetio.py

```diff
@@ -9,11 +9,25 @@
 import os
 from pathlib import Path
 
+import pytest
+
 # How to run this test:
 #
 # LLAMA_STACK_CONFIG="template-name" pytest -v tests/integration/datasetio
 
 
+@pytest.fixture
+def test_dataset(llama_stack_client):
```

Comment:
- can we call this something not with a `test_` prefix because that is how tests are typically marked. 
- also we call `unregister()` manually with the "test_dataset" argument but `register_dataset()` is not passed that argument?

Reply:
renamed fixture & register dataset with explicit id